### DISCUSSION TITLE
Company information and contacts via GitHub pages

### DISCUSSION THREAD

**Discussion by decadenza** [community/community#59893] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am unsure if using a GitHub page to advertise my company would be allowed by the current terms of service.

From the [Prohibited uses](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#prohibited-uses) section I read:

> GitHub Pages is not intended for or allowed to be used as a free web-hosting service to run your online business, e-commerce site, or any other website that is primarily directed at either **facilitating commercial transactions** or providing commercial software as a service (SaaS).

Is just putting my company contacts offering freelance work allowable? Or this may be interpreted as "facilitating commercial transactions"?

Any example of existing company GitHub pages will also be of help.

**Comment by anhtran917** [comment]
It seems that using GitHub Pages to advertise your company and offer freelance work may be allowed.
By examining these examples, you can get an idea of how companies utilize GitHub Pages for various purposes. 

- Facebook: https://facebook.github.io/
- Netflix: https://netflix.github.io/
- Microsoft: https://microsoft.github.io/

However I recommend you contact Github Support Team directly.

**Reply by decadenza** [reply]
Thanks. What's the best way to contact them? Following the help pages it took me only to the community forum.

**Comment by anhtran917** [comment]
You can contact GitHub support directly by following these steps:

- Go to the GitHub Support page: https://support.github.com/

- Scroll down to the bottom of the page and click on the "Contact us" link. This will take you to the GitHub Contact page.

- On this page, you will find a proper case about your inquiry such as "Enterprise Server Support": 
  https://support.github.com/contact/enterprise-by-license
  Go there and fill in the required fields.

Once you have filled out the form, click on the "Send request" button to send your inquiry to GitHub support.
I hope this will help you.
